tools/templates/template.md.php

```diff
@@ -1,35 +1,46 @@
+<?php if (!$versionData) {
+    exit(84);
+} // 0.6 // @phpstan-ignore-line?>
 ---
 layout: default
-title: {{fullVersionName}}
-nav_order: {{order_in_list}}
+title: <?php echo $versionData->fullVersionName; ?>
+
+nav_order: <?php echo $versionData->order_in_list; ?>
+
 parent: Versions
-permalink: docs/versions/{{version}}.html
----
+permalink: docs/versions/<?php echo $versionData->version; ?>.html
+
+--- 
 <div class="callout callout-info" markdown="span">
 Pour être sûr que la machine est bien configurée --> tout doit être en vert sur le script PHP, toutes les checkbox sur ce document cochées.
 </div>
 
-# {{version}}
+# <?php echo $versionData->version; ?>
+
 
 1. TOC
 {:toc}
 
 ## Check
 
-<input type="checkbox"/> Le script [check_{{version}}.php](../versions_tests_scripts/check_{{version}}.php) est à disposition pour check une bonne partie des prérequis.
+<input type="checkbox"/> Le script [check_<?php echo $versionData->version; ?>.php](../versions_tests_scripts/check_<?php echo $versionData->version; ?>.php) est à disposition pour check une bonne partie des prérequis.
 Tous les voyants devraient être verts, attention cependant il est possible d'avoir de faux négatifs (par exemple si la mémoire configurée est supérieure à celle requise).
 
 ## La stack de base
-- <input type="checkbox"/> Debian {{debian_version}}
-- <input type="checkbox"/> PHP {{php_version}}
-- <input type="checkbox"/> Apache {{apache_version}}
-- <input type="checkbox"/> PostgreSQL {{pgsql_version}}
+- <input type="checkbox"/> Debian <?php echo $versionData->debian_version; ?>
+
+- <input type="checkbox"/> PHP <?php echo $versionData->php_version; ?>
+
+- <input type="checkbox"/> Apache <?php echo $versionData->apache_version; ?>
+
+- <input type="checkbox"/> PostgreSQL <?php echo $versionData->pgsql_version; ?>
+
 
 ## Dimensionnement machine
 
 Nous exigeons au minimum :
- * <input type="checkbox"/> {{expected_vcpus}} vCPU
- * <input type="checkbox"/> {{expect_ram_go}}Go RAM
+ * <input type="checkbox"/> <?php echo $versionData->expected_vcpus; ?> vCPU
+ * <input type="checkbox"/> <?php echo $versionData->expect_ram_go; ?> Go RAM
```

Comment:
```suggestion
 * <input type="checkbox"/> <?php echo $versionData->expected_ram_go; ?> Go RAM
```
refactor pour être cohérent dans le nommage des variables avec la ligne du dessus.